### PR TITLE
Fixed Issue 3

### LINKED ISSUE #3: Translation: schedule.html
Translation: schedule.html
-----

* Please translate the content into Japanese and submit the changes as a PR.
* Please assign yourself (or add a comment) to the issue **before** you start translating, so other people know that you are translating.
* Once you have finished translating, send a pull request and reference this issue.

-----
* 内容を日本語にして、翻訳をPRとして送ってください。
* あなたが翻訳し始めたと分かるように、翻訳を**始める前**にこのイシューを自分に割り振って(assign)、あるいはコメントを書いて下さい。
* 翻訳が終わったら、プルリクエストを開いて、このイシューを参照・リンクして下さい。

https://github.com/swcarpentry-ja/2021-04-02-todai-online-ja/blob/24512c032a52aab3822f955bc513ff4ead0b0999/_includes/swc/schedule.html#L1-L47

### PR DESCRIPTION
**Pull request by naganeayu**
Fixed Issue #3

### CODE REVIEW

File: _includes/swc/schedule.html [hunk 1]
```diff
@@ -1,47 +1,48 @@
...
+    <h3>4月7日（水）</h3>
     <table class="table table-striped">
-      <tr> <td>10:00</td>  <td>Welcome</td> </tr>
-      <tr> <td>10:10</td>  <td>Introduction to R and RStudio</td> </tr>
-      <tr> <td>11:00</td>  <td>Break</td> </tr>
-      <tr> <td>11:10</td>  <td>Project Management with RStudio</td> </tr>
-      <tr> <td>11:30</td>  <td>Seeking Help</td> </tr>
-      <tr> <td>12:00</td>  <td>END</td> </tr>
+      <tr> <td>10:00</td>  <td>導入</td> </tr>
+      <tr> <td>10:10</td>  <td>RとRStudioの紹介</td> </tr>
```
Comment (joelnitta) [inline _includes/swc/schedule.html]:
```suggestion
      <tr> <td>10:10</td>  <td>RとRStudioの入門</td> </tr>
```

File: _includes/swc/schedule.html [hunk 2]
```diff
@@ -1,47 +1,48 @@
...
+      <tr> <td>11:00</td>  <td>休憩</td> </tr>
+      <tr> <td>11:10</td>  <td>RStudioのプロジェクト管理</td> </tr>
+      <tr> <td>11:30</td>  <td>ヘルプの参照</td> </tr>
```
Comment (joelnitta) [inline _includes/swc/schedule.html]:
```suggestion
      <tr> <td>11:30</td>  <td>ヘルプ機能</td> </tr>
```

File: _includes/swc/schedule.html [hunk 3]
```diff
@@ -1,47 +1,48 @@
...
-    <h3>April 9</h3>
+    <h3>4月9日（金）</h3>
     <table class="table table-striped">
-      <tr> <td>10:00</td>  <td>Data Structures</td> </tr>
-      <tr> <td>11:00</td>  <td>Break</td> </tr>
-      <tr> <td>11:10</td>  <td>Exploring Data Frames</td> </tr>
-      <tr> <td>12:00</td>  <td>END</td> </tr>
+      <tr> <td>10:00</td>  <td>データ構造</td> </tr>
+      <tr> <td>11:00</td>  <td>休憩</td> </tr>
+      <tr> <td>11:10</td>  <td>データフレームの探索</td> </tr>
```
Comment (joelnitta) [inline _includes/swc/schedule.html]:
```suggestion
      <tr> <td>11:10</td>  <td>データフレームの内容を確認</td> </tr>
```

File: _includes/swc/schedule.html [hunk 4]
```diff
@@ -1,47 +1,48 @@
...
   </div>
   <div class="col-md-6">
-    <h3>April 14</h3>
+    <h3>4月14日（水）</h3>
     <table class="table table-striped">
-      <tr> <td>10:00</td>  <td>Subsetting Data</td> </tr>
-      <tr> <td>11:00</td>  <td>Break</td> </tr>
-      <tr> <td>11:10</td>  <td>Creating Publication-Quality Graphics with ggplot2</td> </tr>
-      <tr> <td>12:00</td>  <td>END</td> </tr>
+      <tr> <td>10:00</td>  <td>データのサブセッティング</td> </tr>
```
Comment (joelnitta) [inline _includes/swc/schedule.html]:
```suggestion
      <tr> <td>10:00</td>  <td>データの部分集合化</td> </tr>
```

File: _includes/swc/schedule.html [hunk 5]
```diff
@@ -1,47 +1,48 @@
...
+      <tr> <td>11:00</td>  <td>休憩</td> </tr>
+      <tr> <td>11:10</td>  <td>ggplot2を使用した出版物並みの品質のグラフィックの作成</td> </tr>
```
Comment (joelnitta) [inline _includes/swc/schedule.html:35]: I actually like this translation more! Perhaps we should change the episode title in the lesson, 「ggplot2 を用いたパブリッシュ出来る品質のグラフィックスの作成」to match?

とりあえずそのままでお願いします。

File: _includes/swc/schedule.html [hunk 6]
```diff
@@ -1,47 +1,48 @@
...
-    <h3>April 16</h3>
+    <h3>4月16日（金）</h3>
     <table class="table table-striped">
-      <tr> <td>10:00</td>  <td>Creating Publication-Quality Graphics with ggplot2 (continued)</td> </tr>
-      <tr> <td>11:00</td>  <td>Break</td> </tr>
-      <tr> <td>11:10</td>  <td>Producing Reports With knitr</td> </tr>
-      <tr> <td>12:00</td>  <td>END</td> </tr>
+      <tr> <td>10:00</td>  <td>ggplot2を使用した出版物並みの品質のグラフィックの作成 (続き)</td> </tr>
+      <tr> <td>11:00</td>  <td>休憩</td> </tr>
+      <tr> <td>11:10</td>  <td>knitrを使用したレポート作成</td> </tr>
```
Comment (joelnitta) [inline _includes/swc/schedule.html]:
```suggestion
      <tr> <td>11:10</td>  <td>knitr によるレポートの作成</td> </tr>
```

**Review by joelnitta — CHANGES_REQUESTED** [review]
I forgot to mention we have already translated the titles for these in the [R lesson](https://swcarpentry-ja.github.io/r-novice-gapminder/ja/). Please use those titles as follows in the suggestions.

**Comment by naganeayu** [comment]
Thank you @joelnitta for suggestions!

**Review by joelnitta — APPROVED** [review]

**Pull request merged**